backend/api/signals.py

```diff
@@ -28,3 +28,41 @@ def handle_stock_post_save(instance: models.Stock, created: bool, **kwargs):
                     stock=instance,
                 ),
             )
+
+        # Add Twitter
+        df = pd.DataFrame()
+        data = services.get_tweets(instance.ticker)
+        df = pd.concat([df, data])
```

Comment:
i think we can just directly assign `df = services.get_tweets(...)`. The reason for me concatenating above is that i have multiple subreddits to get the data from.